### DISCUSSION TITLE
Welcome to supabase-on-do Discussions!

### DISCUSSION THREAD

**Discussion by timothymamo** [digitalocean/supabase-on-do#14] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us what you think of the repo and how you are using it.

  ➡️ You can **convert issues to discussions** either individually or bulk by labels (issues labelled “question” or “discussion”).

**Comment by SteveChurch** [comment]
Thanks for setting up Discussions @timothymamo 🥳 

I'm Steve, a DevOps/Infra Lead for a company called YAGRO based in the UK Agri space. By night I run The Night Project which is a side hustle that houses all of my little side projects and dreams 😄 

This repo is a brilliant baseline for implementing Supabase on Digital Ocean, the alternatives out where are incredibly complicated. You've done a great job of creating something clean and easy to understand. Kudos 💯
